cool_seq_tool/data_sources/feature_overlap.py

```diff
@@ -0,0 +1,180 @@
+
+        # Get chromosome names without prefix and without suffix for alternate
+        # transcripts
+        df["chromosome"] = df["chromosome"].apply(
+            lambda chromosome: chromosome.strip("chr").split("_")[0]
+        )
+        df["chromosome"] = df["chromosome"].astype(str)
+
+        # Convert start and stop to ints
+        df["cds_start"] = df["cds_start"].astype(int)
+        df["cds_stop"] = df["cds_stop"].astype(int)
+
+        # Only retain certain columns
+        df = df[
+            ["type", "chromosome", "cds_start", "cds_stop", "info_name", "gene"]
+        ]
+
+        return df
+
+    def _get_chr_from_alt_ac(self, identifier: str) -> str:
+        """Get chromosome given genomic identifier
+
+        :param identifier: Genomic identifier on GRCh38 assembly
+        :raises FeatureOverlapError: If unable to find associated GRCh38 chromosome
+        :return: Chromosome. 1..22, X, Y. No 'chr' prefix.
+        """
+        aliases, error_msg = self.seqrepo_access.translate_identifier(
+            identifier, "GRCh38"
+        )
+
+        if error_msg:
+            raise FeatureOverlapError(str(error_msg))
+
+        if not aliases:
+            raise FeatureOverlapError(
+                f"Unable to find GRCh38 aliases for: {identifier}"
+            )
+
+        chr_pattern = r"^GRCh38:(?P<chromosome>X|Y|([1-9]|1[0-9]|2[0-2]))$"
+        for a in aliases:
+            chr_match = re.match(chr_pattern, a)
+            if chr_match:
+                break
+
+        if not chr_match:
+            raise FeatureOverlapError(
+                f"Unable to find GRCh38 chromosome for: {identifier}"
+            )
+
+        chr_groupdict = chr_match.groupdict()
+        return chr_groupdict["chromosome"]
+
+    def get_grch38_cds_overlap(
+        self,
+        start: int,
+        end: int,
+        chromosome: Optional[str] = None,
+        identifier: Optional[str] = None,
+        residue_mode: ResidueMode = ResidueMode.RESIDUE,
+    ) -> Optional[Dict]:
+        """Get feature overlap for GRCh38 genomic data
+
+        :param start: GRCh38 start position
+        :param end: GRCh38 end position
+        :param chromosome: Chromosome. 1..22, X, or Y. If not provided, must provide
+            `identifier`. If both `chromosome` and `identifier` are provided,
+            `chromosome` will be used.
+        :param identifier: Genomic identifier on GRCh38 assembly. If not provided,
+            must identifier `chromosome`. If both `chromosome` and `identifier` are
+            provided, `chromosome` will be used.
+        :param residue_mode: Residue mode for `start` and `end`
+        :raise FeatureOverlapError: If missing required fields
+        :return: Feature overlap dictionary where the key is the gene name and the value
+            is the list of CDS overlap (cds_start, cds_stop, overlap_start,
+            overlap_stop). Will return residue coordinates.
```

Comment:
@wesleygoar @austinant Wasn't sure if we wanted to return residue or inter-residue coordinates

Reply:
Hmm good question. We do inter-residue for most of our other operations right? @ahwagner do you want to weight in on this?

Reply:
Yes. I'll wait until @ahwagner replies

Reply:
@austinant did you do this initially with residue based coordinates?

Reply:
@korikuzma sounds good. Pinging @ahwagner again just to make sure he sees this haha

Reply:
Let's use VRS Sequence Location objects for describing regions (so, interresidue). I don't know where SA is at (I think they just created a JSON schema PR today–haven't reviewed) but I know they will be using VRS Sequence Locations to describe features.

Reply:
@wesleygoar i used whatever coords are output by normalizer (for the variant) + whatever coords in the GFF file (for the CDS region)

Reply:
@austinant I can't remember if you had accounted for GFF using 1 based and Variation Normalizer output using inter-residue when getting the overlap, but I addressed this in my latest changes.

Reply:
Updated so that VRS Sequence Locations are used in the response.